reasoner/lube-physical/src/main/scala/com/antgroup/openspg/reasoner/lube/physical/operators/Select.scala

```diff
@@ -21,8 +21,9 @@ import com.antgroup.openspg.reasoner.lube.physical.rdg.RDG
 final case class Select[T <: RDG[T]: TypeTag](
     in: PhysicalOperator[T],
     orderedFields: List[Var],
-    as: List[String])
+    as: List[String],
+    isDistinctGet: Boolean)
     extends PhysicalOperator[T] {
-  def row: T#Records = in.rdg.select(orderedFields, as)
+  def row: T#Records = in.rdg.select(orderedFields, as, isDistinctGet)
```

Comment:
there is no In RDG, it is not necessary to add "distinct" to the select interface; "distinct" should be placed within the row interface.

Reply:
eg, in.rdg.select(orderedFields, as).distinct()